eng/build-utils.ps1

```diff
@@ -279,7 +279,7 @@ function Make-BootstrapBuild([switch]$force32 = $false) {
   $projectPath = "src\NuGet\$packageName\$packageName.Package.csproj"
   $force32Flag = if ($force32) { " /p:BOOTSTRAP32=true" } else { "" }
 
-  Run-MSBuild $projectPath "/restore /t:Pack /p:DotNetUseShippingVersions=true /p:InitialDefineConstants=BOOTSTRAP /p:PackageOutputPath=`"$dir`" /p:EnableNgenOptimization=false $force32Flag" -logFileName "Bootstrap" -configuration $bootstrapConfiguration
+  Run-MSBuild $projectPath "/restore /t:Pack /p:RoslynEnforceCodeStyle=false /p:UseRoslynAnalyzers=false /p:DotNetUseShippingVersions=true /p:InitialDefineConstants=BOOTSTRAP /p:PackageOutputPath=`"$dir`" /p:EnableNgenOptimization=false $force32Flag" -logFileName "Bootstrap" -configuration $bootstrapConfiguration
```

Comment:
> /p:UseRoslynAnalyzers=false

Are we sure we want to make this change? @mavasani have we ever caught bugs by running analyzers in the bootstrap build?

Reply:
100% sure. All of this code is immediately built again with analyzers enabled. Running analyzers during bootstrap build is just burning CPU cycles and making it harder to turn on warn as error. Which is precisely how we got into this state to begin with.